Source/ARTRealtimeChannel.m

```diff
@@ -835,13 +841,16 @@ - (void)onMessage:(ARTProtocolMessage *)pm {
         }
 
         _lastPayloadMessageId = msg.id;
+        
+        // RTL15b
+        self.channelSerial = pm.channelSerial;
```

Comment:
I think `channelSerial` will only be set on the `ProtocolMessage` rather than the messages it contains

Reply:
I also realized that I hadn't set this on attached state - it should now be there d183678a010c14693e791be1fc5d290fb34539eb